docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -122,7 +122,7 @@ When using the `deno-runtime` runner to define a task,
 the task is coded in Deno-script
 (which is mostly the same as JavaScript and TypeScript)
 and executed in the
-[Deno](https://deno.com/) runner,
+[Deno](https://deno.land/manual@v1.36.2/runtime) runner,
```

Comment:
```suggestion
[Deno](https://deno.land/manual) runner,
```
this should always redirect to the latest version so that we do not need to change the link